rollup.config.js

```diff
@@ -13,7 +13,7 @@ import sass from 'sass'
 import analyze from 'rollup-plugin-analyzer'
 
 const extensions = ['.js', '.jsx', '.ts', '.tsx']
-const maxBytes = 5e6 // if > 5MB total, then error
+const maxBytes = 6e6 // if > 5MB total, then error
 const onAnalysis = ({ bundleSize }) => {
   if (bundleSize < maxBytes) return
   console.log(`Bundle size exceeds ${maxBytes} bytes: ${bundleSize} bytes`)
```

Comment:
I'll make a ticket to add this step to the new esbuild bundle. I'm wondering how the analysis gets this number, though, since after tree-shaking/minification we're typically under 2MB for JS and 0.5 MB for CSS